cdk.context.json

```diff
@@ -0,0 +1,40 @@
+{
+  "projectName": "OSML",
+  "account": {
+    "id": "409719124294",
+    "region": "us-west-2",
+    "prodLike": false
+  },
+  "vpc": {
+    "deploy": true,
+    "config": {
+      "VPC_NAME": "OSML-VPC-TEST"
```

Comment:
do we want to include this in the guidance repo? I understand you probably want to include some examples for the customer but some customers will be deploying this out of the box and won't change this value.

Reply:
I will update the context.json to be a more generic boilerplate file.